.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "useTabs": true,
```

Comment:
Oof. Don't use tabs. They look different on different envs and OSes. Use 2 spaces for JS as a rules of thumb.